siliconcompiler/fpgas/lattice_ice40.py

```diff
@@ -30,6 +30,10 @@ def setup(chip):
 
         fpga.set('fpga', part_name, 'vendor', vendor)
         fpga.set('fpga', part_name, 'lutsize', lut_size)
+        fpga.set('fpga', part_name, 'var', 'flop_async_set', 'None')
+        fpga.set('fpga', part_name, 'var', 'flop_async_reset', 'None')
+        fpga.set('fpga', part_name, 'var', 'flop_enable', 'None')
+        fpga.set('fpga', part_name, 'var', 'legalize_flops', 'None')
```

Comment:
I think this would be better as a list of ff features, instead of a bunch of TF flags.
The presence to this key would also indicate if the legalization would occur.